tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -3229,6 +3229,26 @@ TEST (tensor_stream_test, filter_properties_1)
   EXPECT_STREQ (str, "ANY");
   g_free (str);
 
+  /* sub-plugins */
+  g_object_get (filter, "sub-plugins", &str, NULL);
+  /* custom / custom-easy are always available */
+  EXPECT_TRUE (strstr (str, "custom") != NULL);
+  EXPECT_TRUE (strstr (str, "custom-easy") != NULL);
+#ifdef ENABLE_TENSORFLOW_LITE
+  EXPECT_TRUE (strstr (str, "tensorflow-lite") != NULL);
+#endif
+  g_free (str);
```

Comment:
Q. Can NULL be returned in the 3233 line?

Reply:
Currently no, but I will add this case before checking the sub-plugins.
ASSERT_TRUE (str != NULL);

Reply:
g_free does not require NULL checking.